api/api.go

```diff
@@ -78,6 +79,16 @@ func (t *TokenSigner) init() {
 		return
 	}
 	t.privateKey = privateKey
+	// public key
+	publicKeyData, e := os.ReadFile(t.jwtConfig.RSAPublicKeys[0])
```

Comment:
I am assuming `t.jwtConfig.RSAPublicKeys` is guranteed to have a single entry?

Reply:
Yes it must have one entry if RS256 is configured or else startup will fail.